src/utils/utils.ts

```diff
@@ -271,7 +271,7 @@ const generateValueWithAddedSuggestion = (
     generatePlainTextPart(`${isInsertSpaceToNextPart ? ' ' : ''}${currentPart.text.substring(newMentionPartPosition.end)}`),
 
     ...parts.slice(currentPartIndex + 1),
-  ]);
+  ]) + ' ';
```

Comment:
can remove this if you want, but seems like standard mention behavior so user doesn't have to type the space themself